### PR TITLE
Fixes #641 implement custom tab service

### LINKED ISSUE #641: Report issues website opens in browser
**Actual Behaviour**

When we click on report issues it opens up in browser which is slow.

**Expected Behaviour**

We should use custom tabs to open the website so that it happens quickly.

**Steps to reproduce it**

Open the app and click on Report issues from the drawer.

**Would you like to work on the issue?**

Yes!

### PR DESCRIPTION
**Pull request by nikit19**
Fixes issue #641 

Changes: Report issues page will now open in chrome custom tabs

Screenshots for the change: 
screenshot_20170928-171515

### CODE REVIEW

File: app/src/main/java/org/fossasia/pslab/activity/MainActivity.java [hunk 1]
```diff
@@ -57,6 +58,8 @@
     View navHeader;
     private ImageView imgProfile;
     private TextView txtName;
+    /**** CustomTabService*/
+    private CustomTabService cts;
```
Comment (viveksb007) [inline app/src/main/java/org/fossasia/pslab/activity/MainActivity.java]: Please rename the variable and give name that makes some sense like `customTabService` or `issueTrackerTab`.

File: app/src/main/java/org/fossasia/pslab/others/CustomTabService.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+public class CustomTabService {
+    private CustomTabsClient mCustomTabsClient;
+    private CustomTabsSession mCustomTabsSession;
+    private CustomTabsServiceConnection mCustomTabsServiceConnection;
+    private CustomTabsIntent mCustomTabsIntent;
+
+    private Activity activity;
+
+
+    public CustomTabService (Activity act) {
```
Comment (viveksb007) [inline app/src/main/java/org/fossasia/pslab/others/CustomTabService.java]: rename `act` and remove extra lines.

**Comment by nikit19** [comment]
@viveksb007 is it fine now?

**Comment by CloudyPadmal** [comment]
Hi @nikit19 ! Do we need a whole class to call the method to load the URL? I guess this do the job!
```java
String url = "https://github.com/fossasia/pslab-android/issues";
CustomTabsIntent.Builder builder = new CustomTabsIntent.Builder();
CustomTabsIntent customTabsIntent = builder.build();
customTabsIntent.launchUrl(this, Uri.parse(url));
```

**Comment by nikit19** [comment]
@CloudyPadmal  yes we can do it like that but there are two advantages of making a whole class
1 Acc to the official docs it is a way to "Warm up Chrome to make pages load faster" 
https://developer.chrome.com/multidevice/android/customtabs
2 This way it can be easily used in future too.

Initially I did it exactly how you have mentioned but it was not smooth.

**Pull request merged**